bbdb-vcard.el

```diff
@@ -541,13 +541,26 @@ of possible property parameters"
            (bbdb-vcard-elements-of-type name nil structured-p)))))
 
 (defun bbdb-vcard-search (scard type &optional param)
-  (let ((elements (cadr (assoc type scard))))
-    (if param
-        (cl-remove-if 'null
-                      (mapcar (lambda (element)
-                                (cadr (assoc param element)))
-                              elements))
-      elements)))
+  "Search bbdb records from `scard' by `type' and `param'.
+if `type' name match `bbdb-vcard-skip-on-import' or its
+elements (when it is a regexps-list), return `nil'."
+  (let* ((elements (cadr (assoc type scard)))
+         (regexps bbdb-vcard-skip-on-import)
+         (bbdb-import-p
```

Comment:
bbdb-import-p is true if the element should *not* be imported, right? in that case I think it would be less confusing to call the variable 'skip-import-p' or something similar...